tests/toit/utils.toit

```diff
@@ -39,3 +52,56 @@ class ToitExecutable:
       full-command += ["--sdk-dir", sdk-dir_]
     full-command += args
     return pipe.run-program full-command
+
+  fork --with-test-sdk/bool=true args/List -> ForkResult:
+    full-command := [toit_run_, toit-bin-src_]
+    if with-test-sdk:
+      full-command += ["--sdk-dir", sdk-dir_]
+    full-command += args
+    fork-data := pipe.fork
+        true                // use_path
+        // We create a stdin pipe, so that qemu can't interfere with
```

Comment:
Was this lifted from some Qemu test code?

Reply:
yes... removed.
done.